client/src/Movies/MovieList.js

```diff
@@ -32,23 +34,9 @@ export default class MovieList extends Component {
 }
 
 function MovieDetails({ movie }) {
-  const { title, director, metascore, stars } = movie;
   return (
-    <div className="movie-card">
-      <h2>{title}</h2>
-      <div className="movie-director">
-        Director: <em>{director}</em>
-      </div>
-      <div className="movie-metascore">
-        Metascore: <strong>{metascore}</strong>
-      </div>
-      <h3>Actors</h3>
-
-      {stars.map(star => (
-        <div key={star} className="movie-star">
-          {star}
-        </div>
-      ))}
-    </div>
+    <Link to={`/movie/${movie.id}`}>
+      <MovieCard movie={movie} />
+    </Link>
```

Comment:
seems a little silly to call a component inside another component that just passes the param through it, might make more sense just to put this in your MovieList and just get rid of movieDetails